framework/src/media/audio/resample/samplerate.c

```diff
@@ -493,8 +480,15 @@ int src_simple(src_handle_t handle, src_data_t *src_data)
 			   FRAMES_TO_BYTES(src->left_frames));
 		frames_num += src->left_frames;
 
+		//according to the outsize_in_frames, to calculate the adjust_size_in_frames
+		int ratio = new_sample_rate / old_sample_rate;
```

Comment:
"ratio" is already calculated in line 433.
Couldn't this duplication be removed?

Reply:
It has some difference.  
In the 433 line,  it calculate the src_ratio, it is a float value. 
And in this modification , ratio  is a int value ,ex: when  origin_sample_rate is 16000,desired_sample_width is 24000,  the src-ratio is 1.5,  but we need get a ratio value is 2.

Reply:
similar with:
int ratio = lrintf(src_ratio + 0.5);
but there's still some difference:
2.0  -> 2  (if use lrintf, it return 3, not our expected result.)
2.1  -> 3  (lrintf return same value)
2.9 ->  3  (lrintf return same value)

Reply:
I see. Thank you for good explanations.